picky-asn1-x509/src/algorithm_identifier.rs

```diff
@@ -478,6 +490,8 @@ impl<'de> de::Deserialize<'de> for AesParameters {
 #[derive(Clone, Copy, PartialEq, Debug)]
 #[allow(non_camel_case_types)] // 'SHA2_512_224' is clearer than 'SHA2512224' or 'Sha2512224' imo
 pub enum ShaVariant {
+    MD5,
+    SHA1,
```

Comment:
FYI, this is a breaking change because downstream crate will stop compiling if matching on a `ShaVariant` without any wildcard branch.